components/Images/images.css

```diff
@@ -0,0 +1,18 @@
+.container{
+    height: 50%;
+    width: 50%;
+}
+
+
+.img{
+   width: 17rem;
+   height: 17rem;
+   object-fit: cover;
+}
+  .img-rd{
```

Comment:
kee more meaningful name like round instead of rnd,responsive instead of rsv

Reply:
Sure. Thanks for the suggestion.